packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -65,7 +65,7 @@ export const GridCell = React.memo((props: GridCellProps) => {
   const cellRef = React.useRef<HTMLDivElement>(null);
   const apiRef = React.useContext(GridApiContext);
 
-  const cssClasses = clsx(className, `${GRID_CSS_CLASS_PREFIX}-cell${capitalize(align)}`, {
+  const cssClasses = clsx(className, `${GRID_CSS_CLASS_PREFIX}-cell--text${capitalize(align)}`, {
     [`${GRID_CSS_CLASS_PREFIX}-withBorder`]: showRightBorder,
     [`${GRID_CSS_CLASS_PREFIX}-cellEditable`]: isEditable,
```

Comment:
Are we going to have global classes for the data grid, e.g. `.MuiDataGrid-withBorder`? I would use only the ones available in the core.

```suggestion
    [`${GRID_CSS_CLASS_PREFIX}-cell--withBorder`]: showRightBorder,
    [`${GRID_CSS_CLASS_PREFIX}-cell--editable`]: isEditable,
```

Reply:
For the `cellEditable` I agree but not with the `-withBorder`. That class is used on other elements that are not cell, the column header is one for example.